app/controllers/posts_controller.rb

```diff
@@ -8,4 +8,20 @@ def show
     @user = User.find(params[:user_id])
     @post = @user.posts.find(params[:id])
   end
+
+  def new
+    @user = current_user
+  end
+
+  def create
+    @post = current_user.posts.new(params.require(:post).permit(:title, :text))
+    redirect_to user_post_path(current_user, @post) if @post.save
+  end
+
+  def like
+    @post = Post.all.find(params[:id])
+    @user = User.find(params[:user_id])
```

Comment:
- [ ] Same as the comment, you should save the like on behalf of the current_user.

For example
```ruby
current_user.likes.create(post_id: @post.id)
```

Reply:
Hello reviewer! 

Thank for the review!

These are the changes that I made:

- When I create the like, the `author_id` receives the current_user id:

![image](https://user-images.githubusercontent.com/81584449/141354813-19bd2542-3520-4716-a30e-fc183b51be6c.png)

- The same thing with the comment:

![image](https://user-images.githubusercontent.com/81584449/141355030-7972cf52-cf62-49a2-96a9-961c1bdc743a.png)

I just maintain `@user` to return to the correct route when the changes are made.

Again, thank you.